docs: add control plane in-depth guide (#3295)
talos-systems/talos: website/content/docs/v0.9/Learn More/faqs.md

Talos was an automaton created by the Greek God of the forge to protect the island of Crete.
He would patrol the coast and enforce laws throughout the land. We felt it was a fitting name for a security focused operating system designed to run Kubernetes.

## Why does Talos query `pool.ntp.org` on boot even if configured to use a different time server?

> When Talos boots, before config is loaded, Talos tries to do non-blocking attempt to sync the time with the default nameserver (`pool.ntp.org`).

Reviewer: ```suggestion
When Talos boots, before the config is loaded, Talos performs a non-blocking attempt to sync the time with the default nameserver (`pool.ntp.org`).
```

> This initial time sync is required if the node doesn't have RTC or RTC is out of sync: any TLS (e.g. HTTPS) request requires time to be in sync for certificate validation.

Reviewer: ```suggestion
This initial time sync is required if the node doesn't have a  RTC or the RTC is out of sync because TLS (e.g. HTTPS) requires time to be in sync for certificate validation.
```

> This initial time sync is required if the node doesn't have a  RTC or the RTC is out of sync because TLS (e.g. HTTPS) requires time to be in sync for certificate validation.

Reviewer: ```suggestion
This initial time sync is required if the node doesn't have a RTC or the RTC is out of sync because TLS (e.g. HTTPS) requires time to be in sync for certificate validation.
```

> As soon as config is available, Talos starts syncing time with the configured time server.

Reviewer: ```suggestion
As soon as the config is available, Talos starts syncing the time with the configured time server.
```
